components/infobox/wikis/rocketleague/infobox_league_custom.lua

```diff
@@ -0,0 +1,68 @@
+local League = require('Module:VoganRL/Infobox/League')
+local Cell = require('Module:Infobox/Cell')
+local String = require('Module:String')
+
+local RLLeague = {}
+
+function RLLeague.run(frame)
+    League.addCustomCells = RLLeague.addCustomCells
+    League.createTier = RLLeague.createTier
+    League.createPrizepool = RLLeague.createPrizepool
+	League.addCustomContent = RLLeague.addCustomContent
+    return League:createInfobox(frame)
+end
+
+function RLLeague.addCustomCells(company, infobox, args)
+    return infobox
+end
+
+function RLLeague:createTier(args)
+	return Cell	:new('Tier')
+				:options({})
+				:content('Hello')
+end
```

Comment:
still todo, right?

Reply:
yea

Reply:
Will be done in follow-up PR, since this PR is about the general module, not RL specific

Reply:
#221